addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -7108,6 +7147,19 @@ registry.BackgroundShape = SnippetOptionWidget.extend({
             case 'showOnMobile': {
                 return this._getShapeData().showOnMobile;
             }
+            case 'setAnimationDuration': {
+                return params.activeValue || this._getShapeData().animationDuration;
```

Comment:
Check in which case we need this `params.activeValue` part, it should not be necessary.